### PR TITLE
ISSUE-110 Step for Method/changes (Mailbox + Email)

### PR DESCRIPTION
**Pull request by LanKhuat**

### CODE REVIEW

File: src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapMailbox.scala [hunk 1]
```diff
@@ -64,4 +66,24 @@ object JmapMailbox {
...
+  def getNewState(accountId: String = "accountId", oldState: String = "oldState"): HttpRequestBuilder = {
+    JmapHttp.apiCall("mailboxChanges")
+      .body(StringBody(
+        s"""{
+           |  "using": ["urn:ietf:params:jmap:core", "urn:ietf:params:jmap:mail"],
+           |  "methodCalls": [
+           |    ["Mailbox/changes", {
+           |      "accountId": "$${$accountId}",
+           |      "sinceState": "$${$oldState}"
+           |    }, "c1"]
```
Comment (chibenwa) [inline src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapMailbox.scala]: Can we get a back reference here? One to get created mailbox, one to get updated mailboxes?

**Review by chibenwa — COMMENTED** [review]
Looks good so far.

**Review by Arsnael — COMMENTED** [review]
Looks good so far :)

**Review by Arsnael — APPROVED** [review]
Looks good to me :)

**Review by chibenwa — APPROVED** [review]

**Pull request merged**